webapp/src/components/tutorial/TutorialStepCounter.tsx

```diff
@@ -37,7 +39,27 @@ export function TutorialStepCounter(props: TutorialStepCounterProps) {
 
     const stepButtonLabelText = (step: number) => lf("Go to step {0} of {1}", step + 1, totalSteps);
     const backButtonLabel = lf("Go to the previous step of the tutorial.");
-    const nextButtonLabel = lf("Go to the next step of the tutorial.");
+
+    const lastStep = currentStep == totalSteps - 1;
+    const nextButtonTitle = lastStep ? lf("Finish the tutorial.") : lf("Go to the next step of the tutorial.");
+    const nextButtonAction = lastStep ? props.onDone : handleNextStep;
+
+    const nextButton = props.isHorizontal ? (
```

Comment:
Curious if we need to complicate it like this. If the screen is a bit smaller anyway, I'm wondering if we should just make it so that the button to the far right for progressing or finishing the tutorial/skillmap node should just be the icon without text.

Reply:
We need this to be as visible as possible, as its the most important button in the tutorial ui. We've had lots of issues in the past with kids not finding the next button because it was just an arrow and not understanding how to progress